src/cpl/nuopc/lnd_import_export.F90

```diff
@@ -197,8 +197,8 @@ subroutine advertise_fields(gcomp, flds_scalar_name, glc_present, rof_prognostic
 
     ! Fire emissions fluxes from land
     call shr_fire_emis_readnl('drv_flds_in', emis_nflds)
-    if (shr_fire_emis_mechcomps_n .ne. emis_nflds) call shr_sys_abort('ERROR: fire_emis field count mismatch')
-    if (shr_fire_emis_mechcomps_n > 0) then
+    write(6,*)'DEBUG: emis_nflds= ',emis_nflds
```

Comment:
This write to unit 6 is suspicious. Should this be writing to iulog?

Reply:
No - this line should be removed. thanks for catching this. I'll fix this and push it back.